README.md

```diff
@@ -49,6 +49,12 @@ The default duration is *60 seconds*, and the default interval matches it. So gP
 
 * `--no-java`, `--no-python`: Disable the runtime-specific profilers of Java and/or Python, accordingly.
 
+* `--perf-mode`: Controls the global perf strategy. Must be one of the following options:
```

Comment:
There's existing reference in the footnotes to frame pointer, please fix it (refer to the new `--perf-mode` parameter)